### PR TITLE
Ruby: Refactor the generated path and regenerate clients.

### PR DESCRIPTION
**Pull request by landrito**
Uses changes made to googleapis/googleapis made in https://github.com/googleapis/googleapis/pull/420

I completely deleted the old directory and regenerated the following apis:
  "datastore"
  "clouddebugger"
  "dlp"
  "errorreporting"
  "iam"
  "language_v1"
  "language_v1beta2"
  "logging"
  "longrunning"
  "monitoring"
  "pubsub"
  "spanner"
  "spanner_admin_instance"
  "spanner_admin_database"
  "speech_v1"
  "speech_v1beta1"
  "trace"
  "videointelligence"
  "vision"

### CODE REVIEW

File: generated/ruby/google-cloud-pubsub/lib/google/cloud/pubsub/v1.rb [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    #
+    #
+    module Pubsub
+      ##
+      # # Google Cloud Pub/Sub API Contents
+      #
+      # | Class | Description |
+      # | ----- | ----------- |
+      # | [PublisherClient][] | Provides reliable, many-to-many, asynchronous messaging between applications.
+ |
```
Comment (geigerj) [inline generated/ruby/google-cloud-pubsub/lib/google/cloud/pubsub/v1.rb]: What's going on with these pipes? Is this a GAPIC bug?

Comment (landrito) [inline generated/ruby/google-cloud-pubsub/lib/google/cloud/pubsub/v1.rb]: Hm looks like a gapic bug. Let me see if its a quick fix.

Comment (landrito) [inline generated/ruby/google-cloud-pubsub/lib/google/cloud/pubsub/v1.rb]: Its a quick fix. Filed this PR: https://github.com/googleapis/toolkit/pull/1452.

**Comment by landrito** [comment]
Now using this change for toolkit to fix the toc. 
https://github.com/googleapis/toolkit/pull/1452

**Comment by landrito** [comment]
Added CI to the PR, looks like we have two last issues to resolve before merging in the CI.

1. Propagate toolkit changes to Longrunning in gax and publish.
2. Fix unit tests for hash changes in gax.

**Comment by landrito** [comment]
Looks like the CI is passing now!

**Pull request merged**